spec/support/table_list.txt

```diff
@@ -332,6 +331,7 @@ users
 vim_performance_operating_ranges
 vim_performance_states
 vms
+volume_mappings
```

Comment:
This looks unrelated, can you remove from this commit?

Reply:
done